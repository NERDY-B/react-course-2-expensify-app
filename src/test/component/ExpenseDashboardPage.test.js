import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('should test expense dash baord page', ()=>{
    const wrapper = <ExpenseDashboardPage />;
    expect(wrapper).toMatchSnapshot();
})