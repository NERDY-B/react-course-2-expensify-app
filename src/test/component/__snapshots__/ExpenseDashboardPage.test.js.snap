// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test expense dash baord page 1`] = `<ExpenseDashboardPage />`;
